src/Symfony/Component/Notifier/Bridge/Twilio/Webhook/TwilioRequestParser.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Twilio\Webhook;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestMatcher\MethodRequestMatcher;
+use Symfony\Component\HttpFoundation\RequestMatcherInterface;
+use Symfony\Component\RemoteEvent\Event\Sms\SmsEvent;
+use Symfony\Component\Webhook\Client\AbstractRequestParser;
+use Symfony\Component\Webhook\Exception\RejectWebhookException;
+
+final class TwilioRequestParser extends AbstractRequestParser
+{
+    protected function getRequestMatcher(): RequestMatcherInterface
+    {
+        return new MethodRequestMatcher('POST');
+    }
+
+    protected function doParse(Request $request, string $secret): ?SmsEvent
+    {
+        // Statuses: https://www.twilio.com/docs/sms/api/message-resource#message-status-values
+        // Payload examples: https://www.twilio.com/docs/sms/outbound-message-logging
+        $payload = $request->request->all();
```

Comment:
shouldn't it be defined something like `v1` , `vX` etc ? so that the payload, if changed on the service side, is still parseable in core?
(unsure if I am explicitly describing the case :s)

Reply:
Not sure I understand. But if the payload changes, we will adapt the code accordingly to support the old and new formats.

Reply:
oki top thx for answering!